src/Aquifer.Jobs/Program.cs

```diff
@@ -15,6 +15,15 @@
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
+    .ConfigureAppConfiguration((context, config) => 
+    {
+        var envName = context.HostingEnvironment.EnvironmentName;
+        var appSettingsPath = Path.GetFullPath("../../appsettings.json");
```

Comment:
This method of determining the location of the appSettings.json file feels terrible but it was the only method that worked. Other attempts using `config.AddJsonFile("appsettings.json", optional: false, reloadOnChange: true);` would look in the `bin/ouput/` directory resulting in a failed state.

Reply:
Did you look at this at all? https://tharaka-madhusanka.medium.com/how-to-inject-appsettings-environment-json-in-azure-functions-4aea2f10e861

Reply:
This won't work after we deploy because the `appsettings.json` will only be in that location on your workstation and won't be included as part of the deployment package.

Try following the steps in [this SO answer](https://stackoverflow.com/a/78120434):
1. Use `config.SetBasePath(context.HostingEnvironment.ContentRootPath)`
2. Ensure the csproj settings are correct for each of the config files (which will copy the new ones into the output directory).

Reply:
> Did you look at this at all? https://tharaka-madhusanka.medium.com/how-to-inject-appsettings-environment-json-in-azure-functions-4aea2f10e861

@jwinston-bn, that example is for the old in-process model and thus is obsolete now.  Different code is need for the new isolated model which is what we are using (and is the future).  This bit me with durable functions until I learned the difference.

The SO post I shared above is from someone trying to convert from in-process to isolated and should work.